src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  width: $card-width;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  font-family: Roboto, serif;
+  font-weight: 400;
+
+  transition: all linear $main-during;
+
+  &:hover {
+    transform: scale(120%);
```

Comment:
The scale transformation value '120%' should be '1.2' instead of '120%' to properly scale the element by 20%.